.github/workflows/ci-cd-pull-request.yml

```diff
@@ -6,7 +6,55 @@ env:
   BASE_IMAGE: ghcr.io/graphprotocol/graph-docs-staging
 
 jobs:
-  build:
+  lint:
+    if: contains(github.head_ref, 'crowdin') == false
+    runs-on: ubuntu-latest
+    steps:
+      - name: Cancel Previous Runs
+        uses: styfle/cancel-workflow-action@0.9.1
+        with:
+          access_token: ${{ github.token }}
+
+      - name: Check out the repo
+        uses: actions/checkout@v4
+
+      - name: Set up env
+        uses: the-guild-org/shared-config/setup@main
+        with:
+          nodeVersion: 20
+          packageManager: pnpm
+
+      - name: Install dependencies
+        run: pnpm install
+
+      - name: Lint
+        run: pnpm lint
+
+  format:
+    if: contains(github.head_ref, 'crowdin') == false
+    runs-on: ubuntu-latest
+    steps:
+      - name: Cancel Previous Runs
+        uses: styfle/cancel-workflow-action@0.9.1
+        with:
+          access_token: ${{ github.token }}
+
+      - name: Check out the repo
+        uses: actions/checkout@v4
+
+      - name: Set up env
+        uses: the-guild-org/shared-config/setup@main
+        with:
+          nodeVersion: 20
+          packageManager: pnpm
+
+      - name: Install dependencies
+        run: pnpm install
```

Comment:
Are we gonna spend more Actions minutes since this now runs 3 times instead of once?